README.md

```diff
@@ -59,6 +59,26 @@ git diff --cached --name-only --diff-filter=ACM -z | xargs -0 dotnet fantomas
 git diff --cached --name-only --diff-filter=ACM -z | xargs -0 git add -p
 ```
 
+Getting "Format on Save" to work correctly in VS Code is not a straightforward task.
+Existing formatting extensions either don't work at all, use an incompatible version of `fantomas`, or ignore the workspace settings.
+Here is a solution that produces results consistent with our CI checks:
+
+1. Install the right version of `fantomas` globally.
+2. Add the `Custom Local Formatters` extension to VS Code.
+3. Configure the extension by adding this entry to the `customLocalFormatters.formatters` array:
+```json
+  {
+      "command": "fantomas --stdout ${file}",
```

Comment:
Speaking from complete ignorance: Can the file be fed to stdin `fantomas --stdout < ${file}` so that we don't need to comment the stdin hack?

Reply:
`fantomas` supports a `--stdin` argument, but it ignores the workspace settings when using it.
I couldn't find another workaround...